### ISSUE TITLE
Day labor form content changes

### ISSUE THREAD

**Issue by desigonz** [cityofaustin/techstack#2626]
**Acceptance criteria**
- [x] Change "Day Labor" to "Request a day laborer" (no. 1 in screenshot)
- [x] Delete no. 2 in screenshot
- [x] Sentence case for section heading: "Employer information"
- [x] Sentence case for section heading: "Job information"
- [x] Sentence case for section heading : "Phone interview"
- [x] Add checkbox and content under "Hourly pay rate" field:
 -- Ask First Workers’ staff for pay rate suggestions based on the type of job or skills required.

- [x] Confirmation page: Update 2nd sentence to say: "We’ll contact you within 24 hours to confirm."

- [x] Confirmation email: Update 2nd sentence to say: "We’ll contact you within 24 hours to confirm."

![image.png](https://images.zenhubusercontent.com/5a31a50b8a75884b908b0ea6/d914f6e0-70dc-46c3-acae-778b44d8ce91)

**Comment by LauraTrujillo** [comment]
@desigonz copy changes added! Jose shared it with his manager, Derrick, but he doesn't anticipate any other changes.

**Comment by desigonz** [comment]
🙏thanks @LauraTrujillo!

**Comment by Niicck** [comment]
Reviewable at https://opo.austintexas.io/request-day-laborer/2626-day-labor-content/form

**Comment by desigonz** [comment]
A few more updates: 
- [ ] `<title>` should match `h1`, so "Request a day laborer"
- [ ] @Niicck can you look into how messy/difficult it would be to skip the review page? Screenshot of current review page below.


![image](https://user-images.githubusercontent.com/5839508/61906296-ab401b00-aef0-11e9-818e-b52fc771d054.png)

**Comment by desigonz** [comment]
Also I separately submitted a bug issue for the calendar validation: #2642.

**Comment by desigonz** [comment]
Apologies @Niicck—I think we forgot to QA the confirmation email, and we have a new addition per a conversation on Slack—so we have a few more copy changes/additions for ya. Still waiting on @LauraTrujillo to add copy for the review page heading.

**Acceptance criteria"
- [ ] / / @LauraTrujillo, add your copy for the above question here  / / 
- [ ] Add helper text to the “Transportation” field that reads: “The Day Labor Center is located at 4916 North IH-35 Service Road. If your job site is further than 10 miles from the Day Labor Center, we recommend paying an additional $10 to go towards gas money for your worker.”
- [ ] Change "I'd like a suggestion from the First Workers' staff." to "I'd like a suggestion from the Day Labor Center's staff." (for consistency in how we call the center)
- [ ] Confirmation email: Change "Office of TBD" to "First Workers Day Labor Center"
- [ ] Confirmation email: lowercase "information" in "Employer information"
- [ ] Confirmation email: lowercase "information" in "Job information"
- [ ] Confirmation email: see my suggestion in the last screenshot below for an example of how to format the hourly pay rate in a slightly more reader-friendly way.

![image](https://user-images.githubusercontent.com/5839508/61969701-c10d1900-afa0-11e9-8e24-d9e5998fe26e.png)
![image](https://user-images.githubusercontent.com/5839508/61969865-23feb000-afa1-11e9-972b-4277e7b99261.png)

**Comment by LauraTrujillo** [comment]
@desigonz @Niicck I'm so embarrassed- I'm sorry I didn't QA the confirmation email. :( 
I updated the earlier comment with a title for the accordion.

**Comment by desigonz** [comment]
No worries @LauraTrujillo! I should have included it on the QA checklist. Thanks for getting that copy to us :)

**Comment by desigonz** [comment]
I didn't get the confirmation email sent to my inbox.

**Acceptance criteria**
- [ ] On form, add helper text to the “Transportation” field that reads: “The Day Labor Center is located at 4916 North IH-35 Service Road. If your job site is further than 10 miles from the Day Labor Center, we recommend paying an additional $10 to go towards gas money for your worker.”
- [ ] On review page, change "Review Form Submission" to "Day labor request"
- [ ] On confirmation email, let's take out the hyperlink to "Learn more about..." for now (that page isn't up in alpha, it's just linking to a google doc)  
- [ ] If email is included, send confirmation email
- [ ] Confirmation email: Change "Office of TBD" to "First Workers Day Labor Center"
- [ ] Confirmation email: lowercase "information" in "Employer information"
- [ ] Confirmation email: lowercase "information" in "Job information"
- [ ] Confirmation email: see my suggestion in the last screenshot below for an example of how to format the hourly pay rate in a slightly more reader-friendly way.

![image](https://user-images.githubusercontent.com/5839508/61969701-c10d1900-afa0-11e9-8e24-d9e5998fe26e.png)
![image](https://user-images.githubusercontent.com/5839508/61969865-23feb000-afa1-11e9-972b-4277e7b99261.png)

**Comment by easherma** [comment]
https://opo.austintexas.io/request-day-laborer/2626-day-labor-content/form should be up to date on copy changes. Working on the backend/email next

**Comment by easherma** [comment]
All ready for QA!

**Comment by annieszot** [comment]
Thanks @easherma ! I'll tackle QA for the English form :)

**Comment by desigonz** [comment]
- [ ] @annieszot to QA English
- [ ] @desigonz to QA Spanish

**Comment by desigonz** [comment]
@easherma Looks like Spanish translations aren't in??

**Comment by annieszot** [comment]
English Day Labor form QA'd on Chrome browser version 75.0
Acceptance criteria that was met is checked off above. 
A question about colons - they're not on the review page but are on the confirmation email. 
Keep?  
2 bugs reported: #2739 and #2740 
@desigonz Here's a screenshot of the reader-friendly the hourly pay rate on the confirmation email
![Screen Shot 2019-08-14 at 8 57 49 AM](https://user-images.githubusercontent.com/46934140/63028213-fefcaf00-be73-11e9-90b0-a17033cc0691.png)

**Comment by desigonz** [comment]
@annieszot thanks for your thorough QA! Re: colons—form confirmation emails have never had a rigorous content strategy from the ground up; it was put together quickly for OPO and all we've done is copy edits when necessary. I'll make a new issue for content to rethink this in the future, but let's keep it for now.

@easherma here's what still needs to get done:
- [ ] update contact on confirmation email #2740 
- [ ] input Spanish translations (this doc has all of the translations: https://drive.google.com/open?id=1FX13Af3POcNBI2RmiaBuVwpkwXemETB_)
  - [ ] for form
  - [ ] for confirmation email

**Comment by desigonz** [comment]
@easherma The following still need to be fixed:
- [ ] Fix #2772  
- [ ] On form, fix translations for the following fields/copy:
  - [ ] Your name
  - [ ] City
  - [ ] State
  - [ ] We’ll call you at this number if we have questions.
  - [ ] Email address
- [ ] On confirmation email, delete period on this line:

![image](https://user-images.githubusercontent.com/5839508/63299663-eccba800-c29b-11e9-8f2c-b3523b43f628.png)

**Comment by easherma** [comment]
Ok done FRFR!

**Comment by desigonz** [comment]
@easherma it looks request great. My hugest apologies, I missed one last translation. Can you at that in, and then it's ready to go?

- [ ] In Spanish, "Day labor request" translates to "Solicitud de jornalero"
![image](https://user-images.githubusercontent.com/5839508/63352127-1d582400-c326-11e9-87e4-9fbc3e891c06.png)

**Comment by desigonz** [comment]
Looks great! QA'd
